### DISCUSSION TITLE
How to run trivy in debug or quite mode

### DISCUSSION THREAD

**Discussion by nandusekarv10** [aquasecurity/trivy#921] [Q&A]
I am trying to trivy in debug/quite mode. Help showing message 
`GLOBAL OPTIONS:
   --quiet, -q        suppress progress bar and log output (default: false) [$TRIVY_QUIET]
   --debug, -d        debug mode (default: false) [$TRIVY_DEBUG]`

But I am not able to pass that options. Can someone help me with that.

**Comment by szpak** [comment]
Try to just pass `-d` right after `trivy`, not after `trivy image ...`.

For example:
```
trivy -d image alpine:3.14
```
works fine for me with 0.19.2 and 0.20.0.
